docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -2053,303 +2053,285 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Reason:**
 
-  TODO
+  There is a problem calling a sessionStore _APIMethodname_.
 
   **Action:**
 
-  TODO
+  No action needed usually, but error may be needed for debugging.
+  
+
 
 
+### ZWED0153W
 
-### ZWED0157W":"2 function initLoggerMessages - ERROR - %s
+  WARNING: CLI Argument missing name or has unsupported type _ARG_TYPE_FLAG_/_ARG_TYPE_VALUE_/_ARG_TYPE_JSON_.
 
   **Reason:**
 
-  TODO
+  It throws warning when CLI Argument missing any name or has unsupported type for any one of them _ARG_TYPE_FLAG_/_ARG_TYPE_VALUE_/_ARG_TYPE_JSON_.
 
   **Action:**
 
-  TODO
+  Check any missing argument or unsupported argument.
+
 
 
+### ZWED0154W
 
-### ZWED0158W":"%s
+  WARNING: Unrecognized command: _args_
 
   **Reason:**
 
-  TODO
+  Throws warning error _args_ when command is Unrecognized.
 
   **Action:**
 
-  TODO
+  Check the command once again.
 
 
 
-### ZWED0159W":"Plugin (%s) loading failed. Message: "%s" Successful: %s% (%s/%s) Attempted: %s% (%s/%s)
+### ZWED0155W
+
+   _ErrorMsg_
 
   **Reason:**
 
-  TODO
+  Server throws 500 code with _ErrorMsg_.
 
   **Action:**
 
-  TODO
+  Go through the _ErrorMsg_ for detail to debug.
 
 
 
-### ZWED0160W":"Master cluster not found to save storage for id: %s, value: %s
+### ZWED0156W
+
+  1 function initLoggerMessages - ERROR - _Error_
 
   **Reason:**
 
-  TODO
+  When we encountered an _Error_ ,then 1 function created a logger file with specified message.
 
   **Action:**
 
-  TODO
+  Go through the logger file for detail _Error_ to debug.
+
 
 
+### ZWED0157W
 
-### ZWED0161W":"Master cluster not found to save storage
+  2 function initLoggerMessages - ERROR - _Error_.
 
   **Reason:**
 
-  TODO
+  When we encountered an _Error_ ,then 2 function created a logger file with specified message.
 
   **Action:**
 
-  TODO
+  Go through the logger file for detail _Error_ to debug.
+
 
 
+### ZWED0158W
 
-### ZWED0162W":"Master cluster not found to add storage
+   _ErrorMsg_
 
   **Reason:**
 
-  TODO
+  Server throws 500 code with _ErrorMsg_.
 
   **Action:**
 
-  TODO
+  Go through the _ErrorMsg_ for detail to debug.
+
 
 
+### ZWED0159W
 
-### ZWED0163W":"Master cluster was not found to delete storage for id: %s
+  Plugin (_PluginIdentifier_) loading failed. Message: "_pluginsTotal_" Successful: _pluginsLoaded_% (_pluginsLoaded_/_eventCount_) Attempted: _pluginCount_% (_pluginCount_/_eventCount_)
 
   **Reason:**
 
-  TODO
+  Plugin with _pluginId_ loaded failed. The server attempted to load a total of _pluginsTotal_ with _pluginsLoaded_ plugins already successfully loaded.
 
   **Action:**
 
-  TODO
+  Review _errorMessage_ to see if there is something you can do to fix the error. You may need to contact the plugin developer to find a solution. If you do not need this plugin, it is OK to continue.
 
 
 
-### ZWED0165W":"Unable to retrieve storage data from cluster promise %s
+### ZWED0166W
+
+  Error updating the storage: _Error_
```

Comment:
Oh shoot! This error message reason while technically correct, this error message SUCKS. Storage update failed yes but it doesn't say what the key: value pair was

Nothing to do here I suppose